pinot-common/src/main/java/com/linkedin/pinot/common/partition/PartitionAssignment.java

```diff
@@ -26,9 +26,9 @@
  */
 public class PartitionAssignment {
 
-  private String _tableName;
+  protected String _tableName;
   // TODO: include replica in the key, by creating a class PartitionKey to encapsulate partitionId and replicaNum
```

Comment:
Should we remove this TODO, since you chose the approach of extending PartitionAssignment instead of creating a PartitionKey class? Or do you think it might still be a good idea to create a PartitionKey?

Reply:
I think this approach would be more simple. If we introduced `PartitionKey`, we will need to make the current existing realtime partition assignment compatible (e.g. 1 -> 0_1).